public/javascripts/email-validation.js

```diff
@@ -0,0 +1,24 @@
+$(function() {
```

Comment:
I would suggest cleaning this up - there appears to be a lot more going on than there really is, primarily due to the fact that we've mixed constants in with logic:
```js
const VALIDATION = {
  rules: {
    email: {
      required: true,
      // etc.
    }
  }
}

const submitHandler = function( form ) {
  form.submit();
}

$(document).ready( function() {
  $( '#registration' ).validate( Object.assign( {}, VALIDATION, submitHandler ) );
});
```
Also, I personally prefer the `$(document).ready()` syntax - it feels more explicit to me, and minimizes possible for confusion for folks.

We do not need to use an attribute selector to select the form (`"form[id='registration']"`) - we can use the id selector, as above.